src/main/resources/swagger/api-docs.yaml

```
@@ -3392,6 +3405,22 @@ components:
         userAcceptedVersion:
           type: string
           description: the version of the terms of service the user has accepted
+    TermsOfServiceAdherenceStatus:
+      required:
+        - userId
+        - userHasAcceptedLatestTos
+        - acceptedTosAllowsUsage
+      type: object
+      properties:
+        userId:
+          type: string
+          description: user id terms of service adherence pertains to
+        userHasAcceptedLatestTos:
+          type: boolean
+          description: has the user accepted the latest terms of service?
+        acceptedTosAllowsUsage:
+          type: boolean
+          description: based on terms of service status, can the user use Terra?
```

Just following up on my comment from above:
```suggestion
          description: based on the user's currently accepted terms of service version and when the terms of service were last changed, should the user still be permitted to use the system?
```